test/SparkLendMainnetIntegration.t.sol

```diff
@@ -257,9 +271,9 @@ contract SparkLendMainnetIntegrationTest is Test {
         _triggerUpdate(USDC);
         _triggerUpdate(USDT);
 
+        assertEq(_getBorrowRate(USDC), currentRateUSDC + 0.002000000000000000000000000e27, "after2: USDC mismatch");
         // Almost +0.2%, but utilization is slightly below the kink point
-        assertEq(_getBorrowRate(USDC), currentRateUSDC + 0.001891955013226013824859422e27, "after2: USDC mismatch");
-        assertEq(_getBorrowRate(USDT), currentRateUSDT + 0.001899234005225860815063204e27, "after2: USDT mismatch");
+        assertEq(_getBorrowRate(USDT), currentRateUSDT + 0.001602551612415803144238481e27, "after2: USDT mismatch");
     }
 
     function _getBorrowRate(address asset) internal view returns (uint256) {
```

Comment:
We should move the new tests above these and add a section label to indicate these are helpers